core/common/app.php

```diff
@@ -224,6 +224,7 @@ protected function get_init_settings() {
 			'isRTL' => is_rtl(),
 			'isDebug' => ( defined( 'WP_DEBUG' ) && WP_DEBUG ),
 			'activeModules' => array_keys( $this->get_components() ),
+			'rest_url' => get_rest_url(),
```

Comment:
Put it inside `urls` array

Reply:
Handled.

Reply:
https://github.com/elementor/elementor/pull/11388/commits/bbdfa741e9ca7b8f7231f161928a04c97c012938